build.sbt

```diff
@@ -56,6 +56,8 @@ lazy val root = project.withId("spark-snowflake").in(file("."))
       "org.scalatest" %% "scalatest" % "3.1.1" % Test,
       "org.mockito" % "mockito-core" % "1.10.19" % Test,
       "org.apache.commons" % "commons-lang3" % "3.5" % "provided",
+      // For test to read/write from postgresql
+      "org.postgresql" % "postgresql" % "42.3.6" % Test,
```

Comment:
Add `postgresql` as Test library because I want to add `"unit test: SnowflakeSparkUtils.getNameForLogicalPlanOrExpression for JDBCRelation"`